alacritty/src/input.rs

```diff
@@ -496,12 +506,16 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
             {
                 self.ctx.mouse_mut().block_url_launcher = true;
                 self.on_mouse_triple_click(button, point);
+                // Move vi mode cursor to mouse position
+                if self.ctx.terminal().mode().contains(TermMode::VI) {
+                    // Update vi mode cursor position on triple click
+                    self.ctx.terminal_mut().vi_mode_cursor.point = point;
+                }
```

Comment:
If we do this every time the user clicks, could we do this outside of this match to reduce duplication?